src/test/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientHandlerTest.java

```diff
@@ -60,18 +61,15 @@ public void testCreateClientException() {
     }
   }
 
-  @Test
-  public void testCreateClientOverrideBdecVersion() {
-    // remove bdec version
+  @Ignore
```

Comment:
Let's upgrade the ingest SDK and remove the @ignore

Reply:
And i believe there is another end2end test you need to remove

Reply:
Upgraded sdk, didn't see which e2e test to remove, i'll wait till we see which one fails i guess 😅

Reply:
You can check my PR: https://github.com/snowflakedb/snowflake-kafka-connector/pull/649/files